main/src/cgeo/geocaching/apps/cache/WhereYouGoApp.java

```diff
@@ -20,7 +20,7 @@
 import org.apache.commons.lang3.StringUtils;
 
 public class WhereYouGoApp extends AbstractGeneralApp {
-    private static final Pattern PATTERN_CARTRIDGE = Pattern.compile("(" + Pattern.quote("http://www.wherigo.com/cartridge/details.aspx?") + ".*?)" + Pattern.quote("\""));
+    private static final Pattern PATTERN_CARTRIDGE = Pattern.compile("(https?\\:\\/\\/www\\.wherigo\\.com\\/cartridge\\/details\\.aspx\\?[Cc][Gg][Uu][Ii][Dd]=[-0-9a-zA-Z]*)");
```

Comment:
I think that this should be ok:
`Pattern.compile("(https?" + Pattern.quote("://www.wherigo.com/cartridge/details.aspx?") + "[Cc][Gg][Uu][Ii][Dd]=[-0-9a-zA-Z]*)");`

Reply:
Ok, slightly different than the one I had tried. But this one actually works ;-)
Thanks for pointing me to this, will adapt the PR.